pom.xml

```diff
@@ -22,10 +22,24 @@
             <version>5.0.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.6.0</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.konghq/unirest-objectmapper-jackson -->
+        <dependency>
+            <groupId>com.konghq</groupId>
+            <artifactId>unirest-objectmapper-jackson</artifactId>
```

Comment:
Great job! It is a cool solution, but too complicated.

Try another library from Google (GSON).
https://github.com/google/gson

Reply:
fixed

Reply:
Remove unused dependency `unirest-objectmapper-jackson`

Reply:
fixed